doc/semitrans.xml

```diff
@@ -324,3 +324,29 @@ fail]]></Example>
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="WreathProduct">
+  <ManSection>
+    <Oper Name = "WreathProduct" Arg = "S, G"/>
+    <Returns>The Wreath Product of <A>S</A> and <A>G</A>.</Returns>
+    <Description>
+      For a monoid <A>S</A> and a permutation group <A>G</A>,
+      <C>WreathProduct(<A>S</A>, <A>G</A>)</C> outputs the semigroup Wreath
+      product of S and G in terms of its embedding in the full transformation
+      monoid. For example, the Wreath product of T3, the full transformation
```

Comment:
Please don't use T3 or C2 (this is not a standard convention in the Semigroups  package)